src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml

```diff
@@ -14,40 +16,31 @@ jobs:
       set -x
       df -h
 
-      buildConfig=Release
-      # Check if AzDO substitutes in a build config from a variable, and use it if so.
-      if [ '$(_BuildConfig)' != '$''(_BuildConfig)' ]; then
-        buildConfig='$(_BuildConfig)'
-      fi
-
       officialBuildArgs=
       if [ '${{ and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}' = 'True' ]; then
-        officialBuildArgs='/p:DotNetPublishUsingPipelines=true /p:OfficialBuildId=$(BUILD.BUILDNUMBER)'
+        officialBuildArgs='/p:OfficialBuildId=$(BUILD.BUILDNUMBER)'
       fi
 
       ./build.sh \
         --ci \
-        --configuration $buildConfig \
+        --configuration $(_BuildConfig) \
         --publish \
         -bl \
         $officialBuildArgs \
+        /p:DotNetPublishUsingPipelines=true \
```

Comment:
I don't understand this change.  Do we want to publish always?  They way it was, it only published during official builds.

Reply:
We now need to publish the tarball so that it can be consumed by the legs that build the tarball.

Reply:
I see.  That makes sense.